boards/px4/fmu-v6x/src/board_config.h

```diff
@@ -234,6 +233,11 @@
 #define V6X51   HW_VER_REV(0x5,0x1) // FMUV6X,     BMI388 I2C2    HB Mini Rev 1
 #define V6X53   HW_VER_REV(0x5,0x3) // FMUV6X,     Sensor Set     HB Mini Rev 3
 #define V6X54   HW_VER_REV(0x5,0x4) // FMUV6X,     Sensor Set     HB Mini Rev 4
+#define V6X90   HW_VER_REV(0x9,0x0) //                         Rev 0
+#define V6X0910   HW_VER_REV(0x9,0x10)  // ver 9 with V6X rev 10
```

Comment:
Please add comments that indicate usage  i.e Q or SN etc and if that this is an EEPROM. The test from https://github.com/pixhawk/Pixhawk-Standards/pull/30/files?short_path=f9c27e1#diff-f9c27e14acf8351a026c81b95c297e5fc7ebefc6e4501e12d3514007369b155d should help